html-aam/index.html

```diff
@@ -16676,7 +16710,7 @@ <h3>Acknowledgments</h3>
           Add listing of individuals who have providing meaningful
           contributions through commenting that have impacted the specification.
         --></div>
-        <div data-include="https://rawgit.com/w3c/aria/master/../common/acknowledgements/funders.html" data-include-replace="true"></div>
+        <div data-include="../common/acknowledgements/funders.html" data-include-replace="true"></div>
```

Comment:
This change is also landing here: https://github.com/w3c/aria/pull/2288/files#diff-5c3c8936e81538898c83ae11f5271e9ae9324925713a3b03aca58cbd2a3c3014

so  it might cause a conflict